Yogic.py

```diff
@@ -1 +1,441 @@
+from flask import Flask, request, Response, jsonify
+from mailjet_rest import Client
+from werkzeug.security import generate_password_hash, check_password_hash
+from flask_sqlalchemy import SQLAlchemy
+from flask_migrate import Migrate, MigrateCommand
+from flask_script import Manager
+from flask_jwt_extended import (
+JWTManager, jwt_required, create_access_token,
+get_jwt_identity
+)
+import json
+import os
+from datetime import time, datetime, timedelta
 
+
+
+def authenticate(user, auth):
+    if auth:
+        return user
+
+def identity():
+    pass
+
+#init app
+
+app = Flask(__name__)
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+app.config['SQLALCHEMY_DATABASE_URI'] = "postgres://Mai:Codeordie2019@localhost/yogic" #environment file
```

Comment:
for deployment this needs to be moved to env file

Reply:
done 👍